src/SPC/store/SourcePatcher.php

```diff
@@ -269,7 +269,10 @@ public static function patchSwoole(): bool
         if ($version === '5.1.3') {
             self::patchFile('spc_fix_swoole_50513.patch', SOURCE_PATH . '/php-src/ext/swoole');
         }
+        // remove when https://github.com/swoole/swoole-src/pull/5848 is merged
         self::patchFile('swoole_fix_date_time.patch', SOURCE_PATH . '/php-src/ext/swoole');
+        // remove when https://github.com/swoole/swoole-src/pull/5847 is merged
+        self::patchFile('swoole_fix_odbclibs.patch', SOURCE_PATH . '/php-src/ext/swoole');
```

Comment:
Is this patch available for latest swoole version only?

Reply:
I've added >6.0 < 6.1 version constraints. Let's hope they merge the fixes in.

Reply:
Btw 5.x does not even work with php 8.4.